redash/query_runner/snowflake.py

```diff
@@ -64,16 +64,27 @@ def determine_type(cls, data_type, scale):
 
     def _get_connection(self):
         region = self.configuration.get("region")
+        account = self.configuration["account"]
 
         # for us-west we don't need to pass a region (and if we do, it fails to connect)
         if region == "us-west":
             region = None
 
+        if self.configuration.__contains__("host"):
+            host = self.configuration.get("host")
+        else:
+            if region:
+                host = "{}.{}.snowflakecomputing.com".format(account, region)
+            else:
+                host = "{}.snowflakecomputing.com".format(account, region)
```

Comment:
You're passing two values but using only one?

Reply:
Fixed